src/components/KarakterComponent.jsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import styled from "styled-components";
+import karakterImg from "../shared/assets/pic_nicolet.jpg";
+
+export default function Karaktercomponent() {
+  return (
+    <KarakterWrapper>
+      <KarakterHeader>
+        <KarakterAvatar>
+          <img src={karakterImg} alt="foto van Kim" />
+        </KarakterAvatar>
+        <KarakterHeadline>
+          <dl>
+            <KarakterDataTilte>Kim Schenk</KarakterDataTilte>
+            <KarakterDataData>Frontend Developer @FIOD</KarakterDataData>
+          </dl>
+        </KarakterHeadline>
+      </KarakterHeader>
+
+      <KarakterTable>
+        <Table>
+          <div className="table__row">
+            <div className="row__items">
+              <div>skills</div>
+              <div>kim</div>
+              <div>html</div>
+              <div>css</div>
+              <div>javascript</div>
+              <div>typescript</div>
+            </div>
+          </div>
+
+          <div className="table__column">
+            <div className="column__items">
+              <div>0%</div>
+              <div>25%</div>
+              <div>50%</div>
+              <div>75%</div>
+              <div>100%</div>
+              <div>125%</div>
+              <div>150%</div>
+              <div>175%</div>
+              <div>200%</div>
+              <Bar scoreBar="1/ 9; grid-row: 2;">200%</Bar>
```

Comment:
je geeft hier nu letterlijk meerdere css waardes en properties mee. Het werkt omdat je overal de `;` goed hebt staan, maar dit is niet de bedoeling. Kun je een op basis van een getal of percentage de grid column span uitrekenen in je Bar component? Je wilt als afnemer van dit component niet moeten nadenken over welke css je moet meegeven, maar je wilt een percentage of getal kunnen meegeven aan de hand van de waarde van de progressie van de bar.

Reply:
oooYYHHEEAAA het is gelukt